app/forms/support_interface/conditions_form.rb

```diff
@@ -0,0 +1,78 @@
+module SupportInterface
+  class ConditionsForm
+    class OfferConditionField
+      include ActiveModel::Model
+
+      attr_accessor :id, :text
+
+      validates :text, length: { maximum: 255, too_long: ->(c, _) { "Condition #{c.id + 1} must be %{count} characters or fewer" } }
+    end
+
+    include ActiveModel::Model
```

Comment:
Dup!

Reply:
I think this one is needed because the earlier `include` is for the nested `OfferConditionField` class